website/content/api-docs/secret/databases/index.mdx

```diff
@@ -534,6 +534,11 @@ this in order to know the password.
 - `username` `(string: <required>)` – Specifies the database username that this
   Vault role corresponds to.
 
+- `self_managed_password` `(string)` – <EnterpriseAlert product="vault" inline />
```

Comment:
Sorry to keep nitpicking this PR, but given we don't have plans to add this to any other DBs anytime soon should we move this to the postgres API specific page?

We could do something similar to how we extend the connection config...


```
## Create static role
In addition to the parameters defined by the Database backend's [Create static role](/vault/api-docs/secret/databases#create-static-role) endpoint, this plugin
has a number of parameters to further configure a static role.
```

Reply:
Thanks for the clarification, @schavis! This field is part of the Vault API, so I opted to define it here and noted its limited use to select DBs.